hw10/prime_number.py

```diff
@@ -0,0 +1,13 @@
+def is_prime(number):
+    if number <= 1:
+        return f'Do not try to fool me, enter number from 2 to 1000 inclusive'
+    if number >= 1001:
+        return f'Do not try to fool me, enter number from 2 to 1000 inclusive'
```

Comment:
if number <= 1 or number >= 1001:
        return f'Do not try to fool me, enter number from 2 to 1000 inclusive'